<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTBulletObjectSettingsWidget</class>
 <widget class="QWidget" name="QTBulletObjectSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>337</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>303</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>303</width>
    <height>337</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QTBulletObjectSettingsWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Object Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="objectName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Shape Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="shapeType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Box</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sphere</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="shapeProperties">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="width">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <double>2.400000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="height">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <double>4.800000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Length</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="length">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <double>0.750000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Radius</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="radius">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Motion State</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="rotationX">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="rotationY">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="rotationZ">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="positionX">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="positionY">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="positionZ">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Mass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="mass">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Friction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="friction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Rolling Friction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="rollingFriction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>shapeType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>shapeProperties</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotationY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shapeType</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>width</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>height</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>length</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotationZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotationX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>positionZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mass</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>friction</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rollingFriction</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>90</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radius</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>objectName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QTBulletObjectSettingsWidget</receiver>
   <slot>SettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ShowShapeProperties(QString)</slot>
  <slot>SettingsChanged()</slot>
 </slots>
</ui>
